etcdserver/api/v3rpc/key.go

```diff
@@ -55,6 +64,7 @@ func (s *kvServer) Put(ctx context.Context, r *pb.PutRequest) (*pb.PutResponse,
 		return nil, togRPCError(err)
 	}
 
+	resp.Header = s.newResponseHeader(resp.Header.Revision)
```

Comment:
@xiang90 `s.kv.Put` already populates `Revision` in the header with other member vars zero.
So I just wanted to populate header from `kvServer` and overwrite `resp.Header`. It's ugly, I can
change it. Any suggestion?